integration_test/cloudbuild-integration-test.yaml

```diff
@@ -42,7 +42,9 @@
   id: adapter-main
   entrypoint: 'bash'
   args: [ '-c',
-          "/import/bin/import \
+          "java -Dorg.dcm4che3.imageio.codec.ImageReaderFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageReaderFactory.properties \
+                -Dorg.dcm4che3.imageio.codec.ImageWriterFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageWriterFactory.properties \
+                -cp /app/resources:/app/classes:/app/libs/* com.google.cloud.healthcare.imaging.dicomadapter.ImportAdapter \
```

Comment:
Old entrypoint was a shell script (looked like customized gradle wrapper, though I couldn't find exact code that generated it).
New entrypoint is exactly the command you see above, but I can't just call it via "args:" integration test. I need to go through bash to trap SIGTERM so that adapter step could correctly finish.

Reply:
Got it, thanks.